src/Xabbo/ViewModels/Room/Avatars/RoomAvatarsViewModel.cs

```diff
@@ -184,6 +185,7 @@ avatar.Type is AvatarType.User &&
 
         FindAvatarCmd = ReactiveCommand.Create(FindAvatar, hasSingleContextAvatar);
         CopyAvatarToWardrobeCmd = ReactiveCommand.Create(CopyAvatarsToWardrobe, hasAnyContextUser);
+        TradeUserCmd = ReactiveCommand.Create(TradeUser, hasAnyContextUser);

```

Comment:
just change this to `hasSingleContextUser` to make sure only a single user is selected to trade